modules/bmo-about-the-hostfirmwaresettings-resource.adoc

```diff
@@ -0,0 +1,52 @@
+// This is included in the following assemblies:
+//
+// post_installation_configuration/bare-metal-configuration.adoc
+
+:_content-type: CONCEPT
+[id="about-the-hostfirmwaresettings-resource_{context}"]
+= About the HostFirmwareSettings resource
+
+You can use the `HostFirmwareSettings` resource to retrieve and manage the BIOS settings for a host. When a host moves to the `Ready` state, Ironic reads the host's BIOS settings and creates the `HostFirmwareSettings` resource. The resource contains the complete BIOS configuration returned from the baseboard management controller (BMC).
+
+
+[discrete]
+== The HostFirmwareSettings specification
+
+Whereas, the `firmware` field in the `BareMetalHost` resource returns three vendor-independent fields, the `HostFirmwareSettings` resource typically comprises many BIOS settings of vendor-specific fields per host.
+
+
+.HostFirmwareSettings specification
+[options="header"]
+|====
+|Parameters|Description
+
+a|
+----
+<BIOS_setting_name>
+  attribute_type:
+  allowable_values:
+  lower_bound:
+  upper_bound:
+  min_length:
+  max_length:
+  read_only:
+  unique:
+----
+
+a| The `spec` is a simple map consisting of the BIOS setting name and the limits of the setting. The fields include:
```

Comment:
This has combined the 2 resources - HostFirmwareSettings and FirmwareSchema and they should be separate.
*HostFirmwareSettings* settings is just name/value pairs. The status contains the current settings and the spec contains the desired settings that the user sets
*FirmwareSchema* contains the fields as above that show the type and the limits for each setting. This is in the spec.